### DISCUSSION TITLE
Divide by Zero bug

### DISCUSSION THREAD

**Discussion by RB211** [hillsoftware/sv06#26] [General]
I experienced this the other day on a very long print. I am very happy to see that the cause of it(bug in firmware) was discovered. So the pre-release 60b firmware still has the bug?

**Comment by hillsoftware** [comment]
I have not updated 60b and removed that binary.  v57 is the current non bugged version until I push a new release

**Reply by j-lang** [reply]
Is there a plan to update SV06 firmware with new Marlin Release: https://github.com/MarlinFirmware/Marlin/releases/tag/2.1.2.1?
